src/braket/pennylane_plugin/translation.py

```diff
@@ -25,18 +26,79 @@
     StateVector,
     Variance,
 )
+from braket.schema_common import BraketSchemaBase
 from pennylane.operation import Observable, ObservableReturnTypes, Operation
 
 from braket.pennylane_plugin.ops import PSWAP, XY, YY, CPhaseShift00, CPhaseShift01, CPhaseShift10
 
-
-def supported_operations() -> FrozenSet[str]:
-    """Returns the operations supported by the plugin.
+BRAKET_TO_PENNYLANE_OPERATIONS = {
+    "x": "PauliX",
+    "y": "PauliY",
+    "z": "PauliZ",
+    "h": "Hadamard",
+    "ry": "RY",
+    "rx": "RX",
+    "rz": "RZ",
+    "s": "S",
+    "vi": "SX",
+    "t": "T",
+    "cnot": "CNOT",
+    "cy": "CY",
+    "cz": "CZ",
+    "swap": "SWAP",
+    "cswap": "CSWAP",
+    "ccnot": "Toffoli",
+    "phaseshift": "PhaseShift",
+    "unitary": "QubitUnitary",
+    "amplitude_damping": "AmplitudeDamping",
+    "generalized_amplitude_damping": "GeneralizedAmplitudeDamping",
+    "phase_damping": "PhaseDamping",
+    "depolarizing_channel": "DepolarizingChannel",
+    "bit_flip": "BitFlip",
+    "phase_flip": "PhaseFlip",
+    "qubit_channel": "QubitChannel",
+    "cphaseshift": "ControlledPhaseShift",
+    "cphaseshift00": "CPhaseShift00",
+    "cphaseshift01": "CPhaseShift01",
+    "cphaseshift10": "CPhaseShift10",
+    "iswap": "ISWAP",
+    "pswap": "PSWAP",
+    "xy": "XY",
+    "xx": "IsingXX",
+    "yy": "YY",
```

Comment:
Worth noting that PennyLane has an IsingYY gate in their latest release:
https://pennylane.readthedocs.io/en/latest/code/api/pennylane.IsingYY.html
Do we know what to do if it's not in the latest release yet?

Reply:
We still use the YY gate in the translation. I can take a look into changing that translation

Reply:
Sorry, my comment was a bit confusing (I edited it). There's an IsingYY gate but it's not yet in the latest release, so I suspect we should use the existing YY gate for now until PL version 0.17.

Reply:
Not sure if it affects the calculus, but we are due for a release mid-August 🙂

Reply:
Out of curiosity, why wasn't YY released with the other Ising gates?

Reply:
The XX, YY, and ZZ gates were all contributed PRs as part of UnitaryHACK. Unfortunately, XX and ZZ got merged prior to the end of hackathon deadline, but YY didn't (however the contributor came back a couple of weeks later to finish the PR).